pkg/network/ebpf/probes/probes.go

```diff
@@ -34,9 +34,12 @@ const (
 	// ProtocolClassifierGRPCSocketFilter runs a classification rules for gRPC protocols.
 	ProtocolClassifierGRPCSocketFilter ProbeFuncName = "socket__classifier_grpc"
 
-	// NetDevQueue runs a tracepoint that allows us to correlate __sk_buf (in a socket filter) with the `struct sock*`
+	// NetDevQueue runs a raw tracepoint that allows us to correlate __sk_buf (in a socket filter) with the `struct sock*`
 	// belongs (but hidden) for it.
-	NetDevQueue ProbeFuncName = "tracepoint__net__net_dev_queue"
+	NetDevQueue ProbeFuncName = "raw_tracepoint__net__net_dev_queue"
+
+	// NetDevQueuePre4_17_0 is the tracepoint version of the same probe attach on kernels less than 4.17
+	NetDevQueuePre4_17_0 ProbeFuncName = "tracepoint__net__net_dev_queue__pre_4_17_0"
```

Comment:
```suggestion
	NetDevQueueTracepoint ProbeFuncName = "tracepoint__net__net_dev_queue__pre_4_17_0"
```

Reply:
I suggest changing the ebpf function name too. Most/all the other version-based changes are because of kernel function signature changes, not the presence of a particular program type.